bundle/src/test/java/com/adobe/acs/commons/ondeploy/impl/OnDeployExecutorImplTest.java

```diff
@@ -64,7 +65,7 @@
 
 public class OnDeployExecutorImplTest {
     @Rule
-    public final AemContext context = new AemContext(ResourceResolverType.JCR_MOCK);
+    public final AemContext context = new AemContext(ResourceResolverType.JCR_OAK);
```

Comment:
I needed to switch the mock implementation because JCR Mock doesn't implement revert functionality.